server/issue.go

```diff
@@ -181,6 +178,126 @@ func httpAPIGetCreateIssueMetadata(ji Instance, w http.ResponseWriter, r *http.R
 	return http.StatusOK, nil
 }
 
+func httpAPIAttachCommentToIssue(ji Instance, w http.ResponseWriter, r *http.Request) (int, error) {
+	if r.Method != http.MethodPost {
+		return http.StatusMethodNotAllowed,
+			errors.New("method " + r.Method + " is not allowed, must be POST")
+	}
+
+	api := ji.GetPlugin().API
+
+	comment := &struct {
+		PostId string       `json:"post_id"`
+		Fields jira.Comment `json:"fields"`
```

Comment:
Why do we decode into a jira.Comment here?

Reply:
After coming back to this after a couple of days I realize we only need the post_id and IssueKey from the response and do not need to decode into a jira.comment.  I am making changes that